manifests/mcollective.pp

```diff
@@ -1,7 +1,7 @@
 define clamps::mcollective (
-  $user = $title,
-  $amqpass = 'password',
-  $amqserver = '$::server',
+  $user       = $title,
+  $amqpass    = 'password',
+  $amqservers = [$::servername],
```

Comment:
I am not actually sure what `$::server ` is as its not listed in https://docs.puppetlabs.com/puppet/latest/reference/lang_facts_and_builtin_vars.html#puppet-master-variables

Maybe this has always been declare with the correct value in the param?